processor/src/main/resources/org/mapstruct/ap/internal/model/BeanMappingMethod.ftl

```diff
@@ -10,6 +10,9 @@
     <#nt><@includeModel object=annotation/>
 </#list>
 <#if overridden>@Override</#if>
+<#if deprecated>
+    @Deprecated
+</#if>
```

Comment:
Can we perhaps change this and all the other places to:
```suggestion
<#if deprecated>@Deprecated</#if>
```

Reply:
If the if statements are all on the same line, it will output an extra blank line when @deprecated is not present, so I want to avoid it

Reply:
OK I didn't know that. Good point. In any case, we already agreed for the new approach, so we won't need any changes in the ftl files.